src/simplewallet/simplewallet.cpp

```diff
@@ -933,6 +934,9 @@ simple_wallet::simple_wallet()
   m_cmd_binder.set_handler("stop_mining",
                            boost::bind(&simple_wallet::stop_mining, this, _1),
                            tr("Stop mining in the daemon."));
+  m_cmd_binder.set_handler("set_node",
+                           boost::bind(&simple_wallet::set_node, this, _1),
```

Comment:
Shouldn't there be a command syntax like others?

    set_node <nodeurl>[:<port>]

Reply:
Agreed, added it now.